launcher/Application.cpp

```diff
@@ -897,6 +895,7 @@ bool Application::createSetupWizard()
         return false;
     }();
     bool pasteInterventionRequired = settings()->get("PastebinURL") != "";
+    bool themeInterventionRequired = settings()->get("ApplicationTheme") != "";
     bool wizardRequired = javaRequired || languageRequired || pasteInterventionRequired;
 
     if(wizardRequired)
```

Comment:
This seems weird. `ApplicationTheme` will never be empty, as its default value is `system`.

You probably want to make `ApplicationTheme` empty by default and then change this check to `==`. Also it should be ORed to `wizardRequired`.

Reply:
posted on discord, but for the record also here;
Does this look like what you were thinking of? 
![image](https://user-images.githubusercontent.com/31988415/210424902-9db7a049-f8ac-4901-a822-6adf971db47b.png)